### DISCUSSION TITLE
Welcome to snipe-it Discussions!

### DISCUSSION THREAD

**Discussion by snipe** [snipe/snipe-it#8880] [General]
## 👋 Welcome!
  We’re testing Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jackka** [comment]
Hello Snipe!
Feature request through the issue tracker is for formed questions. 
For other questions Gitter is used. 
What kind of reasons for "Discussions"?

**Reply by snipe** [reply]
Hi @jackka - sorry for the delay - still figuring out how these discussion stuff works :) 

Gitter is really kinda there for folks who want casual support/discussion. The actually Snipe-IT core contributors aren't there that often - and the nature of real-time chat can make it challenging to keep threaded discussions coherent.

We'll still leave Gitter available so the community can have convos, but I guess I'd say that this would be the place for non-urgent, longer-term discussions where the Snipe-IT core team can participate.

**Reply by uberbrady** [reply]
We're not sure yet - depends on how this works out, but we might use this for more free-form discussions, perhaps instead of Gitter? Depends on if this works well or not. So we'll see.

It certainly 'feels' less like something as immediate as Gitter, but better for interactive discussion than a GitHub issue. So I dunno. Let's try it out and find out!

**Reply by uberbrady** [reply]
hahahah, Jinx!

**Comment by hjbreedt** [comment]
Hi 

Can snipe-it be hosted on a WHM shared hosting server, and if so, do you maybe have an installation guide for this ?

**Reply by uberbrady** [reply]
Installing Snipe-IT in shared-hosting environments is pretty tricky. Check out: https://snipe-it.readme.io/docs/requirements for details. But, no, we don't have any specific documentation about WHM.

**Reply by snipe** [reply]
It's only tricky because we require the `DocumentRoot` to be `public` and WHM by default has it set to `public_html` usually. Typically there is an option to change that though, via WHM or cPanel settings on a per site (Snipe-IT in this case) basis.

**Comment by uberbrady** [comment]
We are definitely still learning how to use this new 'discussions' feature - I just wanted to say we're sorry we're not as on top of it as we should be - we're still learning about it.

**Comment by Surya-Shekhar** [comment]
Hi.. Can someone assist me in creating organization structure role wise and give permission of the assets based on roles?

Regards,
Surya

**Comment by kmpc808** [comment]
Hello all!

I am currently hosting snipe-IT through my synology server.

When setting up the container, I used my server's internal IP address as the APP_URL. In hindsight, I feel like I should have used the DDNS instead, so I could access the software from outside my wi-fi.

Unfortunately, I did not have the foresight to do so and have already created plenty of assets and their corresponding QR codes.

I have a strong feeling that I will have to change the APP_URL to my DDNS and re-print out all the assets' QR codes but before doing so, it seemed like a good idea to hop on here and get some advice/suggestions.

Any ideas?

Regards!
